CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+
+# Contribution Rules📚:
+
+- You are allowed to make pull requests. We just merge it ;)
```

Comment:
We don't just merge it. We review and then merge. So, you can remove that sentence and just keep Anyone is allowed to make a PR.